src/core/include/openvino/core/any.hpp

```diff
@@ -264,11 +337,12 @@ class OPENVINO_API Any {
 
     void impl_check() const;
 
-    mutable Base::Ptr _variant_impl;
-
-    Base::Ptr _impl;
+    mutable Base::Ptr _runtime_attribute_impl;
 
 public:
+    Base::Ptr
+        _impl;  ///< Pointer to implementation. TODO: make it private when ngraph::RuntimeAttributeWrapper is deprecated
```

Comment:
when `RuntimeAttributeWrapper ` will be deprecated?

Reply:
moved to private